src/pages/PSC_ManageSoftCredits.page

```diff
@@ -1,8 +1,13 @@
-<apex:page standardController="Opportunity" extensions="PSC_ManageSoftCredits_CTRL" showHeader="true" standardStylesheets="true" docType="html-5.0">
+<apex:page standardController="Opportunity"
+    extensions="PSC_ManageSoftCredits_CTRL"
```

Comment:
The manage Soft Credits has Contact marked as required, which I think is correct since it cannot be empty. The field border is red on error while it is not on production. @mpusto  any objections?

Branch org:
<img width="1415" alt="Screen Shot 2020-08-18 at 4 56 21 PM" src="https://user-images.githubusercontent.com/28743286/90576791-05eb8b00-e174-11ea-962e-d3071c27516e.png">


Main (production) org:
<img width="1416" alt="Screen Shot 2020-08-18 at 4 56 30 PM" src="https://user-images.githubusercontent.com/28743286/90576780-fff5aa00-e173-11ea-8f46-6d1eadb73017.png">

Reply:
@rponti-sforg , the above will be different when compared to an org built of the `master`.